### PR TITLE
Website Version 1

### PR DESCRIPTION
**Pull request by garrying**
This will be deployed by **April 30**. Feedback is welcomed!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Jekyll Related
 _site/
 .sass-cache/
 .jekyll-metadata
+
+# OS X sorry
```
Comment (benhylau) [inline .gitignore:6]: :100:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,18 +4,18 @@ We love pull requests! Our process for accepting changes has a few steps:
 
 1. If you haven't submitted anything before, and you aren't (yet!) a member of our organization, fork and clone the repo:
```
Comment (benhylau) [inline CONTRIBUTING.md]: @dcwalk Didn't we already move this to https://github.com/tomeshnet/documents?

File: _config.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+permalink: pretty
+
+# Asset compiling
+sass:
+  style: :compressed
+
+# Site settings
+title: Toronto Mesh
+email: hi@tomesh.net
+description: >
```
Comment (benhylau) [inline _config.yml]: We need an actual _awesome description_?

File: _config.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  Write an awesome description for your new site here. You can edit this
+  line in _config.yml. It will appear in your document head meta (for
+  Google search results) and in your feed.xml site description.
+baseurl: "" # the subpath of your site, e.g. /blog
+url: "http://tomesh.net" # the base hostname & protocol for your site
+github_username:  tomeshnet
```
Comment (benhylau) [inline _config.yml]: Extra space

File: index.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      {{ vision | markdownify }}
+    </div>
+  </div>
+  <!-- /.intro-header -->
+
+  <div class="banner">
+    <h2>Come Say Hi</h2>
+    <a href="https://civictechto-slack-invite.herokuapp.com/" class="btn btn-default btn-lg">
+      <span class="icon icon--slack">{% include icon-slack.svg %}</span>
+      <span class="service-name"> #torontomesh on CivicTechTO slack</span>
```
Comment (benhylau) [inline index.html]: Proposing to rename slack channel to #tomesh for consistency.

File: _config.yml (hunk 3 above)
Comment (garrying) [inline _config.yml]: @benhylau Maybe a _good_ description for now?

File: CONTRIBUTING.md (hunk 2 above)
Comment (dcwalk) [inline CONTRIBUTING.md]: I just merged it in [here](https://github.com/tomeshnet/documents/blob/master/CONTRIBUTING.md)..._(edit: but have made changes to reflect @garrying added, here is an open PR: [#8](https://github.com/tomeshnet/documents/pull/8))_
The link can be updated and contrib removed... thought I didn't commit either when I did the first pass, looks like they go swept up at some point.

File: _config.yml (hunk 3 above)
Comment (dcwalk) [inline _config.yml]: I think the one-liner might do "We are building a mesh network in Toronto."

File: index.html (hunk 5 above)
Comment (dcwalk) [inline index.html]: :+1: yes please for consistency

File: CONTRIBUTING.md (hunk 2 above)
Comment (garrying) [inline CONTRIBUTING.md]: Updated `readme.md` and removed  `contributing.md`.

File: README.md [hunk 6]
```diff
@@ -25,3 +25,11 @@ $ gem install jekyll
 ```bash
 $ jekyll serve
 ```
+
+A development server will run at `http://localhost:4000/`
+
+## Deployment
+
+Commits and merges into `master` will be deployed automatically to the production web server through webhook posts from GitHub.
```
Comment (dcwalk) [inline README.md:33]: Do we want to talk a little bit more about setup? We need a place to lay out other config stuff (e.g., nginx) but maybe this isn't the right spot

**Comment by dcwalk** [comment]
Looking good :bomb:! Just that done comment RE: deployment but I think that can get resolved via an issue/further documentation

**Comment by benhylau** [comment]
@patcon @garry Can we change Slack info to **#tomesh** then? Both here and on Slack?

**Comment by patcon** [comment]
I think anyone can with:

```
/rename tomesh
```

Anyhow, just did it :)

**Pull request merged**